ui-tests/src/test/java/io/syndesis/qe/pages/integrations/editor/add/connection/actions/fragments/ConfigureAction.java

```diff
@@ -14,7 +14,8 @@
 public class ConfigureAction extends SyndesisPageObject {
 
     private static final class Element {
-        public static final By ROOT = By.cssSelector("syndesis-integration-action-configure");
+        //TODO replace this when another selector will be in place
```

Comment:
again the same question - do we track these todos or we fix it when it breaks?

Reply:
Well this was note for me. It's possible that there won't be another selector in place. Second thing is if we even need root documents validation if we have ID's to all necessary elements